### ISSUE TITLE
Why the commit not analytic correctly ?

### ISSUE THREAD

**Issue by kamontat** [semantic-release/commit-analyzer#102]
When I create pull request from Github, and using squash merges. This commit became to one commit. but the analytic didn't parse the commit correctly ?

[ref] https://circleci.com/gh/kamontat/Portfolio/558 on dump version tab.

**Comment by pvdlg** [comment]
The subject of your commit is `Merge deployment #41 from dev` which is not a valid Angular convention commit it's not detected by **semanticc-release**.

You have a detailed answer to your question there: https://github.com/semantic-release/semantic-release/blob/master/docs/support/troubleshooting.md#squashed-commits-are-ignored-by-semantic-release

**Comment by kamontat** [comment]
I already custom merge parser

```js
"parserOpts": {
    "mergePattern": "/[mM]erge (.*) #(\\d+) from (.*)$/",
    "mergeCorrespondence": ["plain", "id", "source"]
}
```

BUT...
